README.md

```diff
@@ -1,6 +1,21 @@
 # Token Lists
 
-Automativally generate Token Lists
+Automatically generate Token Lists
+
+## How to add an image to the CoW Swap default token list
+
+In order to add a new token to the CoW Swap default token list follow these steps
+
+- Find an icon with reasonable quality/size (~256px, <100kB).
```

Comment:
Why not exactly 256x256? I think most of the current images in the list (if not all have that size exactly).

@fairlighteth might have some extra-requirements (or different requirements. What is a good process here?